ts/DataGrid/DataGridOptions.d.ts

```diff
@@ -21,21 +27,82 @@ import type DataTable from '../Data/DataTable';
  *
  * */
 
+/**
+ * Options to control the content and the user expierence of a grid structure.
+ */
 export interface DataGridOptions {
+
+    /**
+     * Height of each grid cell. This is used to calculate the amount of visible
+     * cells in a container and the size of the scrollbar.
```

Comment:
I wouldn't say it controls the size of the scrollbar.
IMO it should be:
`cells in a container and the size of the scrollable container.`

Reply:
I find your suggestion too technical. In the implementation it might be also related to the container, but this is only done to have a proper size of the scrollbar toggle.